### ISSUE TITLE
make mixer, coin-join sites

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#47]
Wikipedia: [CoinJoin](https://en.wikipedia.org/wiki/CoinJoin)

advantages:
* anonymity
* staking participation currently: only about a quarter of total coins are staking: bounces from 50-100 million GRCs

question:
* legal issues ?

mentioned in [hangout 2](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2016_08_13.MD): 2:41:47

**Comment by grctest** [comment]
Assign to: @caraka ?

Advantages: Disassociating coins w/ an CPID. Currently GRC is very transparent if you register your CPID to the NN & your investor coins are stored in addresses used to stake with the CPID. This could increase voting participation.

**Comment by Erkan-Yilmaz** [comment]
related: BitShares upcoming stealth function

* if it comes, @grctest would make a video for it

**Comment by grctest** [comment]
>related: BitShares upcoming stealth function : if it comes, @grctest would make a video for it

Indeed, BitShares has always had stealth built in, but it has been hidden away in the CLI & thus off-limits to 99% of the Bitshares userbase. When Bitshares Munich successfully implements a simple to use Stealth mechanism, it'll appear as 'unknown user sent unknown user2 an unknown amount of an unknown token' to which tokens can be transfered back to the Gridcoin network.
